monitored_resources.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stackdriver
+
+import (
+	monitoredrespb "google.golang.org/genproto/googleapis/api/monitoredres"
```

Comment:
Preferably monitored resource should be a self-contained object instead of depending on Stackdriver Monitoring class. In Java this is to avoid Stackdriver Trace exporter depending on Stackdriver Monitoring. Not sure if this applies to Go? 

/cc @ramonza @bogdandrutu

Reply:
makes sense. I'll make the change.